aws-amplify/docs
# Remove preloading fonts that weren't being used

#### Description of changes:
- Remove preloading unused fonts

Staging site: https://remove-font-preload.d1ywzrxfkb9wgg.amplifyapp.com/

To test:
1. Check the network tab on a browser's web tools to see if these fonts get loaded. Compare it to the production site and verify that the `AmazonEmber_W_Tg.woff2` and `AmazonEmber_W_Lt.woff2` aren't being loaded

#### Related GitHub issue #, if available:

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?

Which platform(s) are affected by this PR (if applicable)?

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks




      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._
